CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ### 0.8.1 (Next)
 
-* Your contribution here.
+* [#30](https://github.com/dblock/ruby-enum/pull/30): value defaults to key if not specified
```

Comment:
Stick to the format.

```
* [#30](https://github.com/dblock/ruby-enum/pull/30): Default value to key - [@gi](https://github.com/gi).
```

I'll add danger-changelog to lint this for PRs when I have time.